Map/Test/MapTest.cpp

```diff
@@ -1,10 +1,30 @@
+#include "FieldTypes.hpp"
+#include "SquareMap.hpp"
 #include <gtest/gtest.h>
-#include "Map.hpp"
 
 using namespace ::testing;
 
-// TEST(Maptest, GetRoomsShouldReturnOneRoom)
-// {
-//     Map map(1);
-//     ASSERT_EQ(map.getRooms().size(), 1);
-// }
\ No newline at end of file
+TEST(Maptest, quantityOfMapColumnsShouldEqualMapSize)
+{
+    constexpr auto mapSize = 8u;
```

Comment:
to sie powtarza w testach, można to wynieśc do animowego namespace
namespace
{
constexpr auto mapSize = 8u;
}